ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/provision/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-provision-chain.yaml

```diff
@@ -0,0 +1,28 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-provision
+  steps: 
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-credentials-mode
+    - ref: ipi-conf-gcp-xpn-network
+    - ref: ipi-conf-gcp-create-firewall-rules
+    - ref: gcp-provision-xpn-min-perm-passthrough-serviceaccount
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+    - ref: cucushift-installer-check-gcp-credentials-mode
+  env:
+  - name: CREDENTIALS_MODE
+    default: "Passthrough"
+    documentation: CCO mode.
+  - name: COMPUTE_NODES_TAGS
```

Comment:
We do not need this setting, right?

Reply:
The service account in use has minimal permissions, and it is lack of the permissions to create/delete firewall-rules in the shared VPC. In this case, the installer won't create firewall-rules in the shared VPC, and the cluster machines are expected to use the pre-configured firewall-rules, which have targetTags configured. For example, the control-plane machines has the network tag "preserved-ipi-xpn-control-plane", and the access targeting to these machines would be allowed per the pre-configured firewall-rules. 

_FYI_
```
$ gcloud --project openshift-qe-shared-vpc compute firewall-rules list --format="table(creationTimestamp.date('%Y-%m-%d %H:%M:%S'):sort=1,name,targetTags)" --filter="network=installer-shared-vpc AND name~preserved-ipi-xpn" 2> /dev/null
CREATION_TIMESTAMP   NAME                                TARGET_TAGS
2022-10-13 02:39:55  preserved-ipi-xpn-api               ['preserved-ipi-xpn-control-plane', 'preserved-ipi-xpn-compute']
2022-10-13 02:46:23  preserved-ipi-xpn-etcd              ['preserved-ipi-xpn-control-plane']
2022-10-13 02:46:29  preserved-ipi-xpn-control-plane     ['preserved-ipi-xpn-control-plane']
2022-10-13 02:46:35  preserved-ipi-xpn-health-checks     ['preserved-ipi-xpn-control-plane', 'preserved-ipi-xpn-compute']
2022-10-13 02:46:44  preserved-ipi-xpn-internal-cluster  ['preserved-ipi-xpn-control-plane', 'preserved-ipi-xpn-compute']
2022-10-13 02:46:50  preserved-ipi-xpn-internal-network  ['preserved-ipi-xpn-control-plane', 'preserved-ipi-xpn-compute']
2022-10-17 20:45:23  preserved-ipi-xpn-bastion-access    ['preserved-ipi-xpn-bastion']
$

Reply:
Got it. thanks for the explaination.